cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +411,84 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), fmt.Sprintf("was not able to find the xt_u32 machine config %s in the %s machine config pool", mc.Name, mcp.Name))
 		})
 	})
+
+	Context("n3000", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			testclient.Client.List(context.Background(), n3000NodeList)
+			if len(n3000NodeList.Items) < 1 {
+				Skip("Cluster doesn't have intel fpga n3000 nic")
+			}
+		})
+
+		It("should have at least 1 intel n3000 nic", func() {
+			n3000NicExists := false
+		FindN3000Nic:
+			for _, n3000Node := range n3000NodeList.Items {
+				for _, fortville := range n3000Node.Status.Fortville {
+					for _, nic := range fortville.NICs {
+						if strings.Contains(nic.Name, "N3000") {
+							n3000NicExists = true
+							break FindN3000Nic
+						}
+					}
+				}
+			}
+			Expect(n3000NicExists).To(BeTrue(), "No Intel fpga n3000 NICs are presented on the n3000nodes")
+		})
+
+		It("should have a ready deployment n3000-controller-manager", func() {
```

Comment:
should have a ready deployment for the OpenNESS Operator for Intel FPGA PAC N3000 (Programming) operator